src/server/apps/authserver/Main.cpp

```diff
@@ -118,8 +118,14 @@ int main(int argc, char** argv)
     }
 
     // Initialize the database connection
+   try {
     if (!StartDB())
         return 1;
+} catch (const std::exception& ex) {
+    LOG_ERROR("server.authserver", "Database initialization failed: {}", ex.what());
+    return 1;
+}
```

Comment:
hello and thanks for your contribution! can you please fix the code indentation?